src/devices/src/virtio/block/device.rs

```diff
@@ -490,6 +510,26 @@ pub(crate) mod tests {
         assert_eq!(b.interrupt_evt.read().unwrap(), 1);
     }
 
+    #[test]
+    fn test_disk_backing_file_helper() {
```

Comment:
A negative test for disk properties would make sense here.

Reply:
What kind of negative test do you have in mind? I'm coming up blank :see_no_evil:

Reply:
Done.